Jenkinsfile

```diff
@@ -0,0 +1,82 @@
+#!groovy
+
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2020
+ */
+
+
+node('ibm-jenkins-slave-dind') {
+  def lib = library("jenkins-library").org.zowe.jenkins_shared_library
+
+  def pipeline = lib.pipelines.nodejs.NodeJSPipeline.new(this)
+
+  pipeline.admins.add("stevenh")
+
+  pipeline.setup(
+    packageName: 'org.zowe.sample-node-api',
+    nodeJsVersion: 'v10.18.1',
+    github: [
+      email                      : lib.Constants.DEFAULT_GITHUB_ROBOT_EMAIL,
+      usernamePasswordCredential : lib.Constants.DEFAULT_GITHUB_ROBOT_CREDENTIAL,
+    ],
+    artifactory: [
+      url                        : lib.Constants.DEFAULT_LFJ_ARTIFACTORY_URL,
+      usernamePasswordCredential : lib.Constants.DEFAULT_LFJ_ARTIFACTORY_ROBOT_CREDENTIAL,
+    ],
+    pax: [
+      sshHost                    : lib.Constants.DEFAULT_PAX_PACKAGING_SSH_HOST,
+      sshPort                    : lib.Constants.DEFAULT_PAX_PACKAGING_SSH_PORT,
+      sshCredential              : lib.Constants.DEFAULT_PAX_PACKAGING_SSH_CREDENTIAL,
+      remoteWorkspace            : lib.Constants.DEFAULT_PAX_PACKAGING_REMOTE_WORKSPACE,
+    ],
+    installRegistries: [
+      [
+        email                      : lib.Constants.DEFAULT_LFJ_NPM_PRIVATE_REGISTRY_EMAIL,
+        usernamePasswordCredential : lib.Constants.DEFAULT_LFJ_NPM_PRIVATE_REGISTRY_CREDENTIAL,
+        registry                   : lib.Constants.DEFAULT_LFJ_NPM_PRIVATE_REGISTRY_INSTALL,
+      ]
+    ],
+    publishRegistry: [
+      email                      : lib.Constants.DEFAULT_LFJ_NPM_PRIVATE_REGISTRY_EMAIL,
+      usernamePasswordCredential : lib.Constants.DEFAULT_LFJ_NPM_PRIVATE_REGISTRY_CREDENTIAL,
+    ],
+    // FIXME: ideally this should set to false (using default by remove this line)
+    ignoreAuditFailure            : true,
+    // FIXME: npm version in ibm-jenkins-slave-dind is too old, doesn't support "npm ci"
+    alwaysUseNpmInstall           : true
```

Comment:
Any possibilities we can remove these two tweaks?

I believe `alwaysUseNpmInstall` can be safely removed since we do a `npm install npm` in base build image to make sure it's always latest version.